ci/scripts/build.sh

```diff
@@ -2,9 +2,9 @@
 
 cwd=$(pwd)
 
-export GOPATH=$cwd/go
+export GOPATH=$cwd
```

Comment:
Maybe we can rename this var? (as it is no longer GOPATH because we are using go modules),
I was doing the following in other repos:
```
cwd=$(pwd)
pushd $cwd/florence
...
```